code/obj/item/device/radio.dm

```diff
@@ -475,7 +475,11 @@ var/list/headset_channel_lookup
 				R.show_message(thisR, 2)
 
 		if (length(heard_normal))
-			rendered = "[part_a][real_name ? real_name : M.real_name][part_b][M.say_quote(messages[1])][part_c]"
+			var/prep_name = "[real_name ? real_name : M.real_name]"
+			if(ishuman(M))
+				if (M:vdisfigured)
+					prep_name = "Unknown"
```

Comment:
```suggestion
			if(M.vdisfigured)
				prep_name = "Unknown"
```
no need to check for human or use the dreaded `:` operator since `vdisfigured` is defined on mob